doc/rtd/topics/analyze.rst

```diff
@@ -167,85 +171,88 @@ consumed for other reporting needs. Each element in the list is a boot entry.
 
 .. code-block:: shell-session
 
-  $ cloud-init analyze dump
-  [
-  {
-    "description": "starting search for local datasources",
-    "event_type": "start",
-    "name": "init-local",
-    "origin": "cloudinit",
-    "timestamp": 1567057578.037
-  },
-  {
-    "description": "attempting to read from cache [check]",
-    "event_type": "start",
-    "name": "init-local/check-cache",
-    "origin": "cloudinit",
-    "timestamp": 1567057578.054
-  },
-  {
-    "description": "no cache found",
-    "event_type": "finish",
-    "name": "init-local/check-cache",
-    "origin": "cloudinit",
-    "result": "SUCCESS",
-    "timestamp": 1567057578.056
-  },
-  {
-    "description": "searching for local data from DataSourceNoCloud",
-    "event_type": "start",
-    "name": "init-local/search-NoCloud",
-    "origin": "cloudinit",
-    "timestamp": 1567057578.147
-  },
-  {
-    "description": "found local data from DataSourceNoCloud",
-    "event_type": "finish",
-    "name": "init-local/search-NoCloud",
-    "origin": "cloudinit",
-    "result": "SUCCESS",
-    "timestamp": 1567057578.55
-  },
-  {
-    "description": "searching for local datasources",
-    "event_type": "finish",
-    "name": "init-local",
-    "origin": "cloudinit",
-    "result": "SUCCESS",
-    "timestamp": 1567057578.979
-  },
-  {
-    "description": "searching for network datasources",
-    "event_type": "start",
-    "name": "init-network",
-    "origin": "cloudinit",
-    "timestamp": 1567057582.814
-  },
-  {
-    "description": "attempting to read from cache [trust]",
-    "event_type": "start",
-    "name": "init-network/check-cache",
-    "origin": "cloudinit",
-    "timestamp": 1567057582.832
-  },
-  ...
-  {
-    "description": "config-power-state-change ran successfully",
-    "event_type": "finish",
-    "name": "modules-final/config-power-state-change",
-    "origin": "cloudinit",
-    "result": "SUCCESS",
-    "timestamp": 1567057594.98
-  },
-  {
-    "description": "running modules for final",
-    "event_type": "finish",
-    "name": "modules-final",
-    "origin": "cloudinit",
-    "result": "SUCCESS",
-    "timestamp": 1567057594.982
-  }
-  ]
+    $ cloud-init analyze dump
```

Comment:
Same comment about `Example Output:` line between command and example output.

Reply:
+1